Resources/public/js/views/ez-contenteditview.js

```diff
@@ -313,7 +352,16 @@ YUI.add('ez-contenteditview', function (Y) {
                         version: this.get('version')
                     });
                 }
-            }
+            },
+
+            /**
+             * The language code in which the content is edited.
+             *
+             * @attribute languageCode
+             * @type {String}
+             * @required
+             */
+            languageCode: {},
```

Comment:
`_getViewParameters` is changed already: https://github.com/ezsystems/PlatformUIBundle/pull/310/files#diff-322d0e70fded2e094b6aea22bb14a63cR281
But I forget to test it :) So now there is test for it and `active` attribute check on `languageCodeChange` event handling.